### DISCUSSION TITLE
Release Notes CovPass Version 1.26.1 (hotfix)

### DISCUSSION THREAD

**Discussion by MoritzIBM** [Digitaler-Impfnachweis/covpass-ios#195] [Announcements]
**Release Notes CovPass Version 1.26.1**

Hi folks,

release 1.26.1 (hotfix) has just gone live.

Bug fixing for certificate renewals: 💉

- Correction and enhancement of misleading text (iOS(Android)
- Renewal of 1/2 certificates is now also possible for iOS users if there is no newer certificate  (e.g., 2/2, 3/3) in the app 

Many thanks to the community for your immediate feedback and support - take care!
Team CovPass and CovPassCheck! 🧑‍⚕️ 👩‍⚕️ 🤖

**Comment by Ein-Tim** [comment]
Hey @MoritzIBM 

Will you create a GitHub release (which includes the updated source code) soon?

**Reply by MoritzIBM** [reply]
Hi @Ein-Tim, on it and it is coming soon - apologies for the delay.
